src/sentry/digests/backends/redis.py

```diff
@@ -70,8 +71,11 @@ class RedisBackend(Backend):
 
     """
 
+    cluster: rb.Cluster
+
     def __init__(self, **options: Any) -> None:
         self.cluster, options = get_cluster_from_options("SENTRY_DIGESTS_OPTIONS", options)
+        assert isinstance(self.cluster, rb.Cluster)
```

Comment:
this can be:

```python
cluster, options = get_cluster_from_options("SENTRY_DIGESTS_OPTIONS", options)
assert isinstance(cluster, rb.Cluster)
self.cluster = cluster
```

and then you can remove the class-level annotation